### DISCUSSION TITLE
Error install package azure-iot-sdk-c

### DISCUSSION THREAD

**Discussion by sergiosolorzano** [microsoft/vcpkg#27200] [Q&A]
I'm following guide https://vcpkg.io/en/getting-started.html to install vcpkg in Win10 machine.
When I reach step ./vcpkg.exe install azure-iot-sdk-c:x64-windows I get error:

PS C:\dev\vcpkg> ./vcpkg.exe install azure-iot-sdk-c:x64-windows
Computing installation plan...
error: this vcpkg instance requires a manifest with a specified baseline in order to interact with ports. Please add 'builtin-baseline' to the manifest or add a 'vcpkg-configuration.json' that redefines the default registry.

Please help. Thank you!

**Comment by JackBoosY** [comment]
The error message said:
`Please add 'builtin-baseline' to the manifest or add a 'vcpkg-configuration.json' that redefines the default registry.`
